subsys/settings/src/settings_tfm_psa.c

```diff
@@ -15,48 +15,67 @@
 /* TF-M config file containing ITS_MAX_ASSET_SIZE */
 #include <config_base.h>
 
-#include "settings_its_priv.h"
+#include "settings_tfm_psa_priv.h"
 
 LOG_MODULE_DECLARE(settings, CONFIG_SETTINGS_LOG_LEVEL);
 
 K_MUTEX_DEFINE(worker_mutex);
 static struct k_work_delayable worker;
 
-static struct setting_entry entries[CONFIG_SETTINGS_TFM_ITS_NUM_ENTRIES];
+static struct setting_entry entries[CONFIG_SETTINGS_TFM_PSA_NUM_ENTRIES];
 static int entries_count;
 
-static int settings_its_load(struct settings_store *cs, const struct settings_load_arg *arg);
-static int settings_its_save(struct settings_store *cs, const char *name, const char *value,
+static int settings_psa_load(struct settings_store *cs, const struct settings_load_arg *arg);
+static int settings_psa_save(struct settings_store *cs, const char *name, const char *value,
 			     size_t val_len);
 
-static const struct settings_store_itf settings_its_itf = {
-	.csi_load = settings_its_load,
-	.csi_save = settings_its_save,
+static const struct settings_store_itf settings_psa_itf = {
+	.csi_load = settings_psa_load,
+	.csi_save = settings_psa_save,
 };
 
-static struct settings_store default_settings_its = {.cs_itf = &settings_its_itf};
+static struct settings_store default_settings_psa = {.cs_itf = &settings_psa_itf};
+
+#if defined(CONFIG_SETTINGS_TFM_PSA_BACKEND_PS)
+#include <psa/protected_storage.h>
+
+#define SETTINGS_PSA_CHUNK_SIZE PS_MAX_ASSET_SIZE
+#define SETTINGS_PSA_SET psa_ps_set
+#define SETTINGS_PSA_GET psa_ps_get
+#elif defined(CONFIG_SETTINGS_TFM_PSA_BACKEND_ITS)
```

Comment:
Add some newline for readability?
```suggestion
#define SETTINGS_PSA_GET psa_ps_get

#elif defined(CONFIG_SETTINGS_TFM_PSA_BACKEND_ITS)
```